src/lib/optimizer/strategy/in_expression_rewrite_rule.hpp

```diff
@@ -28,7 +28,7 @@ class InExpressionRewriteRule : public AbstractRule {
 
   // With the auto strategy, IN expressions whose input has more than MIN_INPUT_ROWS_FOR_DISJUNCTION are rewritten
   // into disjunctive predicates.
-  constexpr static auto MIN_INPUT_ROWS_FOR_DISJUNCTION = 1,000,000f; // TODO Benchmark smaller numbers
+  constexpr static auto MIN_INPUT_ROWS_FOR_DISJUNCTION = 1'000'000.f; // TODO Benchmark smaller numbers
```

Comment:
Just a small request: please think shortly about running this rule with SF1000.
We made several assumptions only for SF 10 in the past. We should at least consider large SFs to determine cases where rules “break”.